docs/basic-features/image-optimization.md

```diff
@@ -51,6 +50,39 @@ function Home() {
 export default Home
 ```
 
+## Image Imports
+
+You can `import` images that live in your project. (Note that `require` is not supported—only `import`.)
+
+With direct `import`s, `width`, `height`, and `blurDataURL` will be automatically provided to the image component. Alt text is still needed separately.
+
+```js
+import Image from 'next/image'
+import profilePic from '../public/me.png'
+
+function Home() {
+  return (
+    <>
+      <h1>My Homepage</h1>
+      <Image
+        src={profilePic}
+        alt="Picture of the author"
+        // width={500} automatically provided
+        // height={500} automatically provided
+        // blurDataURL="data:..." automatically provided
+        // Optionally allows to add a blurred version of the image while loading
+        // placeholder="blur"
+      />
+      <p>Welcome to my homepage!</p>
+    </>
+  )
+}
+```
+
+For dynamic images you have to provide `width`, `height` and `blurDataURL` manually.
```

Comment:
This could be more clear if you define "dynamic image" which really means "src is a string"

Reply:
```suggestion
For images where you’re passing a URL to `src`, you have to provide `width`, `height` and `blurDataURL` manually.
```